test/e2e/specs/blocks/shared/block-smoke-testing.ts

```diff
@@ -48,6 +48,10 @@ export function createBlockTests( specName: string, blockFlows: BlockFlow[] ): v
 			await editorPage.visit( 'post' );
 		} );
 
+		it( 'Enter post title', async () => {
+			await editorPage.enterTitle( `${ specName } - ${ DataHelper.getDateString( 'ISO-8601' ) }` );
+		} );
+
```

Comment:
This is a QoL change, meant to make identifying the post easier - currently no block smoke testing flows generate titles, so it's all "Untitled".

Reply:
Nice, great idea!